### PR TITLE
Issuetemplates

### PR DESCRIPTION
**Pull request by luwol03**
| Status  | Type  |
| :---: | :---: |
| Ready | Refactor |

## Description

**I will merge this directly into main due to the limitation, that the templates are used from main.**

- improved issue templates
- added localazy badge

## How Has This Been Tested?
 
fork / locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking change which cleans up / improves existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

**Review by zikowang — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -14,24 +14,9 @@
...
 <!--- Bonus points for GIFS --->
 
-## Types of changes
-
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Enhancement (non-breaking change which cleans up / improves existing functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
```
Comment (noctera) [inline .github/pull_request_template.md:33]: Why did you remove them completely?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -23,12 +23,20 @@ A clear and concise description of what you expected to happen.
...
- - npm version:
- - node version:
- - electron verison:
- - electron-forge version:
+**Please complete the following information:**
+<!-- 1. navigate to help -> about -->
+<!-- 2. press copy to copy the full version info to your clipboard -->
+<!-- 3. paste version in here -->
+```
+Version:
```
Comment (noctera) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Is the formatting intended compared to before

Comment (luwol03) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: yes, I just copied it from the help > about tab.

File: .github/pull_request_template.md (hunk 1 above)
Comment (noctera) [reply .github/pull_request_template.md:33]: Oh I see

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (noctera) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Ok

**Review by noctera — APPROVED** [review]
